db/migrate/20191002195108_add_nt_info_to_alignment_config.rb

```diff
@@ -0,0 +1,5 @@
+class AddNtInfoToAlignmentConfig < ActiveRecord::Migration[5.1]
+  def change
+    add_column :alignment_configs, :s3_nt_info_db_path, :text
```

Comment:
please add a comment such as ```

add_column :alignment_configs, :s3_nt_info_db_path, :text, comment: "Used for Reference Length and Reference Name  in the coverage viz. Column added 2019-10-03."
```

Reply:
Hey Greg, sorry I missed this comment. Will remember to add comments going forward.